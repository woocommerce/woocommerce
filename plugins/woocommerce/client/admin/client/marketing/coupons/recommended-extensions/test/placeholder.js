/**
 * External dependencies
 */
import { render } from '@testing-library/react';
import { createElement } from '@wordpress/element';

/**
 * Internal dependencies
 */
import { RecommendedExtensionsPlaceholder } from '..';

describe( 'RecommendedExtensionsPlaceholder', () => {
	test( 'should render a default placeholder', () => {
		const { container } = render( <RecommendedExtensionsPlaceholder /> );
		expect( container ).toMatchSnapshot();
	} );
} );
