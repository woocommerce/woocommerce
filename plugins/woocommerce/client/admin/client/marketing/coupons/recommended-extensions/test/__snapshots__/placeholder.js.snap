// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecommendedExtensionsPlaceholder should render a default placeholder 1`] = `
<div>
  <div
    aria-hidden="true"
    class="is-loading woocommerce-marketing-recommended-extensions-item"
  >
    <div
      class="woocommerce-admin-marketing-product-icon is-placeholder"
    />
    <div
      class="is-loading woocommerce-marketing-recommended-extensions-item__text"
    >
      <h4
        aria-hidden="true"
        class="is-placeholder"
      />
      <p>
        <span
          class="is-placeholder"
        />
        <span
          class="is-placeholder"
        />
        <span
          class="is-placeholder"
        />
      </p>
    </div>
  </div>
</div>
`;
